bin/snapshots

```diff
@@ -83,7 +83,7 @@ echo --- twirl $TWIRL_VERSION
 echo
 
 cd $DEPLOY/twirl
-build $TWIRL_VERSION ++2.11.11 ++2.12.2 publish ++2.10.6 publish plugin/publish
```

Comment:
twirl is failing when we try to build with 2.10.6 and as a result none of the snapshots are being published. 
https://github.com/playframework/twirl/pull/190/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR104